pyro/distributions/torch_wrapper.py

```diff
@@ -13,13 +13,12 @@ class TorchDistribution(Distribution):
     `torch.distributions.Distribution <http://pytorch.org/docs/master/_modules/torch/distributions.html#Distribution>`_
     """
 
-    def __init__(self, torch_dist, x_shape, event_dim, batch_size=None, log_pdf_mask=None):
+    def __init__(self, torch_dist, x_shape, event_dim, log_pdf_mask=None, *args, **kwargs):
```

Comment:
Can we remove the `*args, **kwargs`? Otherwise `batch_size` may be silently ignored.

Reply:
Actually, your comment above is related to this. This is basically to absorb the `sample_shape` argument, but I think that it will be cleaner to remove that in `random_primitive` than adding args/kwargs here. Will change.

Reply:
Thanks, I just want to make sure that users will see errors in their existing code that uses `batch_size` in distribution constructors. If the constructor silently swallows the arg then users will only later see a mysterious broadcasting error, or worse a scaled gradient with no error.

Reply:
Makes sense, will change.